apps/daimo-mobile/src/App.tsx

```diff
@@ -33,19 +29,6 @@ export default function App() {
   // White background to avoid between-tab flicker
   let theme = DefaultTheme;
   theme = { ...theme, colors: { ...theme.colors, background: color.white } };
-
-  // Remove splash screen
-  useEffect(() => {
-    const nowS = now();
-    if (account == null || nowS - account.lastBlockTimestamp < 60 * 10) {
-      console.log(`[APP] removing splash now, first render`);
-      SplashScreen.hideAsync();
-    } else {
-      console.log(`[APP] removing splash after sync or 2s, whichever is first`);
-      setTimeout(() => SplashScreen.hideAsync(), 2000);
```

Comment:
hmmm now that i'm seeing this, what if we just delete the redundant 300ms timeout below and reduce the max-wait-time here?

if this `useEffect` is working correctly, it should already hide the splash screen as soon as the account is updated

Reply:
This will result in blank screen because the splash screen is hidden before navigator is rendered. That's why `SplashScreen.hideAsync()` is in navigator now

Reply:
the purpose of the using `SplashScreen.preventAutoHideAsync()` was to wait for the initial `getAccountHistory()` to return, so that we don't flash an incorrect/stale balance